### DISCUSSION TITLE
line-numbers-init for code chunks in reveal.js

### DISCUSSION THREAD

**Discussion by maxpel** [quarto-dev/quarto-cli#7490] [Feature Requests]
### Description

It would be great to have the option to start code line numbering at a different value than 1, especially when working with reveal.js slides as sometimes we have to split up longer code chunks over multiple slides when presenting.

A chunk option like `code-line-numbers-init` would solve that, for example:

`#| code-line-numbers-init: 30`

That suggestion follows the reveal.js plugin: https://github.com/mikemiles86/reveal-line-numbers

**Comment by mcanouil** [comment]
Closing in favour of:
- #1454
